### ISSUE TITLE
how to run this project?

### ISSUE THREAD

**Issue by ggsnpdsn** [phthinh/OFDM_802_16#2]
I want to test the actual effect of this project. Are there any instructions or steps on how to build a hardware platform，thanks

**Comment by phthinh** [comment]
Hi, the hardware system consists of Verilog files and some Xilinx IP cores, e.g., FFT, Complex Multiplier, Cordic. To re-build the system, it needs to create a Vivado project and add the needed Verilog files and pertinent configured IP cores into the project. It is a [Tcl script](https://github.com/phthinh/OFDM_802_16/blob/master/OFDM_RX_802_16/script/make_project.tcl) (resp. [another script](https://github.com/phthinh/OFDM_802_16/blob/master/OFDM_TX_802_16/script/make_project.tcl)) doing the process of creating the project for the Receiver (resp. Transmitter) systems automatically. In Windows OS, you can use a [.bat script](https://github.com/phthinh/OFDM_802_16/blob/master/OFDM_RX_802_16/proj.bat) to invoke the Tcl script. Note that, it is needed to create a 'work' directory and change the Vivado install path accordingly to your computer.

**Comment by ggsnpdsn** [comment]
Is zynq7030 + adi9361 or zynq7100 + adi9361 suitable for hardware? Or do you have any recommended hardware?

**Comment by phthinh** [comment]
Hi, I prefer a bigger FPGA. But be careful to select an FPGA board, Like, adi9361 may require FMC connectors.

**Comment by ggsnpdsn** [comment]
Zynq7000+adi9361 can buy a ready-made development board, but I don't know how to run this project

**Comment by phthinh** [comment]
Basically, the project can not be viewed as a completed system/prototype. It might be considered as an acceleration IP  core to support the computation of OFDM modulation for the baseband modulation of a wireless communication system. The sub-system in the project could be integrated into a bigger system as on one end, it plugs into an SoC consisting of a processor which processes data frames in higher layers; on the other end, it needs some glue logic components to connect to a front-end RF. Mostly, the project focuses on sonly optimizing the hardware and power consumption of the OFDM modulation (particularly in synchronization) instead of building a complete prototype. So, if you want to build a prototype of an OFDM-based communication system, I guess there are many implementation works that need to be done.

**Comment by ggsnpdsn** [comment]
Yes, I thought it was an 802.16 system that could work directly, so I wanted to run it on a zynq+9361 circuit board. It seems difficult to make it work in TDMA mode. I want to build a TDMA wireless communication system through it. Any good suggestions?
